frontend_react/src/app/routes/_app.tsx

```diff
@@ -27,7 +26,11 @@ function RouteComponent() {
               to="/dashboard"
               className="flex flex-1 flex-col items-center gap-1 text-[0.625rem]"
             >
-              <LucideHome />
+              <RiveIcon
+                className="size-8"
+                iconType="Home"
+                isActive={currentPath === '/dashboard'}
+              />
               Home
```

Comment:
[fyi] こんな書き方ができます
今の書き方だと、
* dashboardと完全一致の場合しか取れない
* せっかくtanstackで型安全なルーティングしてるのに、パス名をハードコードする必要がある
のであんまりお勧めしないかも

```suggestion
              {(state) => (
                <>
                  <RiveIcon
                    className="size-8"
                    iconType="Home"
                    isActive={state.isActive}
                  />
                  Home
                </>
              )}
```

https://tanstack.com/router/latest/docs/framework/react/api/router/LinkPropsType#children

Reply:
ありがとうございます！修正しました！